Orange/data/io.py

```diff
@@ -133,6 +133,27 @@ def guess_data_type(orig_values):
 
 def sanitize_variable(valuemap, values, orig_values, coltype, coltype_kwargs,
                       domain_vars, existing_var, new_var_name, data=None):
+    def reorder_values():
```

Comment:
I would rather have this in a separate commit. I assume you just moved the code into a separate method to make code more readable (and radon happy) but I might be wrong.

Reply:
I agree and I'd usually do so. But if you don't insist I wouldn't waste time tearing this apart. It's highly unlikely that anybody would ever again see this. Splitting this PR into two commits would be useful if somebody would want to revert just a part of this PR. Probably won't happen.

Reply:
Leave it as it is. I commented after a spent a could of minutes figuring out what you changed,only to realise it was just refactoring.

Reply:
Leave it as it is. I wrote the comment after I spent a few minutes figuring out what changed in the code, only to find out that it was just some refactoring.